### PR TITLE
Skill effects are functions instead of numbers

### PR DESCRIPTION
**Pull request by jsfehler**
Also, add strength and endurance stats, although they don't do much right now.

### CODE REVIEW

File: skills.js [hunk 1]
```diff
@@ -1,54 +1,85 @@
 'use strict';
 const constants = require('./constants');
 
+// Effect functions for each skill
+
+//TODO: Real calculations for effects
+var useSlice = function (user, target) {
```
Comment (euoia) [inline skills.js]: Can you use `const` instead of `var` please? It's more consistent with the rest of the code base.

File: skills.js [hunk 2]
```diff
@@ -1,54 +1,85 @@
...
+
+var useDrain = function (user, target) {
+    target.strength -= 1 + (user.strength / 2);
+}
+
 module.exports = [
-  {
+{
     name: 'slice',
     type: constants.ATTACKING,
```
Comment (euoia) [inline skills.js:38]: I think you can remove the `type` property now and the corresponding constants.

**Comment by euoia** [comment]
Looks good, just a couple of comments.

File: skills.js (hunk 2 above)
Comment (jsfehler) [inline skills.js:38]: Probably better to keep them for now. Although they're not used in battle, they still define what the general purpose of the skill is.

Comment (euoia) [inline skills.js:38]: If you like, but generally speaking I think it is confusing to have code (or data in this case) that doesn't do anything.

**Comment by euoia** [comment]
LGTM feel free to merge.

**Pull request closed without merging**